content/docs/replenishment/purchase orders/index.md

```diff
@@ -0,0 +1,537 @@
++++
+title = "Purchase Orders"
+description = "Requesting stock from external suppliers."
+date = 2025-09-26
+updated = 2025-09-26
+draft = false
+weight = 44
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Intro
+
+## Viewing Purchase Orders
+
+### Go to the Purchase Orders Menu
+
+First you need to have purchase orders enabled for your store.
+
+1. Go to the `Manage` menu in the navigation panel
+2. Click on `Stores`, then select your store
+3. In the `Preferences` section, enable `Use procurement functionality`
+
+![Go to Purchase Orders](images/po_navigation.png)
+
+Now you will be able to access the **Purchase Order** menu:
+
+1. Go to the `Replenishment` menu in the navigation panel
+2. Click on `Purchase Orders`
```

Comment:
having the image where it is makes it look like it is part of the preferences task. I'd suggest moving it down:

```suggestion
Now you will be able to access the **Purchase Order** menu:

![Go to Purchase Orders](images/po_navigation.png)

1. Go to the `Replenishment` menu in the navigation panel
2. Click on `Purchase Orders`

```

Reply:
I've also resized the image, it's 50% smaller and looks better I think